packages/core/src/routes/connector.ts

```diff
@@ -277,6 +277,14 @@ export default function connectorRoutes<T extends AuthedRouter>(router: T) {
 
       await deleteConnectorById(id);
 
+      try {
+        const logtoConnector = await getLogtoConnectorById(id);
```

Comment:
The connector has been deleted, this will fail?

Reply:
Fixed the logic.